src/Uno.UI/UI/Xaml/Controls/CalendarDatePicker/CalendarDatePicker.Android.cs

```diff
@@ -0,0 +1,205 @@
+﻿#if XAMARIN_ANDROID
+using Uno.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Windows.UI.Xaml.Controls
+{
+
+
+	public partial class CalendarDatePicker : Control
+	{
+		public event EventHandler<CalendarDatePickerDateChangedEventArgs> DateChanged;
```

Comment:
You had to add an exclusion because the event you copied from the generated types is not the same, therfore creating a breaking change. It should be like this: 
```suggestion
public  event TypedEventHandler<CalendarDatePicker, CalendarDatePickerDateChangedEventArgs> DateChanged;
```

Once changed you can remove the exclusions, as they won't be needed anymore.

Reply:
Ok.
I just copied this (and much of CalendarDatePicker.Android.cs) from DatePicker :)